src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountTransactionSupervisor.kt

```diff
@@ -291,6 +291,8 @@ internal class SubaccountTransactionSupervisor(
 
     internal fun cancelTriggerOrdersWithClosedOrFlippedPositions() {
         val subaccount = stateMachine.state?.subaccount(subaccountNumber) ?: return
+        if (stateMachine.state?.marketsSummary?.marketIds()?.isEmpty() != false) return
```

Comment:
meant to catch true or null

Reply:
Nit: There is `isNullOrEmpty()` that can make this more readable